src/select.rs

```diff
@@ -87,3 +92,40 @@ impl<'a> TElement<'a> for &'a ElementData {
         }
     }
 }
+
+impl<'a> Node<'a> {
+    pub fn css(&'a self, css_str: &str) -> FilterNodes<'a, Descendants<'a>> {
```

Comment:
Nokogiri is wrong :) `select` is for *Select*ors. If/when XPath support is added, the method can be named `xpath`.